doc/release/RELEASE-NOTES.md

```diff
@@ -26,11 +26,13 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe bugfix here.*
 
 ### What's new
+* [OSDEV-1105](https://opensupplyhub.atlassian.net/browse/OSDEV-1105) Contribution. Allow commas in list name and update error message.
 * [OSDEV-272](https://opensupplyhub.atlassian.net/browse/OSDEV-272) Facility Claims Page. Implement ascending/descending and alphabetic sort on FE.
 
 ### Release instructions:
 * *Provide release instructions here.*
 
+
```

Comment:
remove extra space

Reply:
I guess we have to keep 2 empty lines after each release section.

Reply:
Yes